### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Build Stage:
```
Comment (xansec) [inline Dockerfile:1]: Can we add a package stage?

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Namespaced project name that the target belongs to
```
Comment (xansec) [inline Mayhemfile:1]: What's the purpose of `filepath` here? Also, please omit image and use generic project name.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Mayhemfile (hunk 2 above)
Comment (bmwtsn098) [reply Mayhemfile:1]: This program takes YML files and if the file doesn't have the proper extension it ignores it. I used the parameter to name the file with the appropriate extension and redid the file the way you fixed my other project.

**Comment by bmwtsn098** [comment]
I made the requested changes. Let me know if everything looks good. Thanks

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**